scalafix-core/shared/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -125,16 +134,16 @@ case class ExplicitResultTypes(
     }
 
     ctx.tree.collect {
-      case t @ Defn.Val(mods, _, None, body)
-          if isRuleCandidate(t, mods, body) =>
+      case t @ Defn.Val(mods, Seq(Pat.Var(name)), None, body)
```

Comment:
nitpick, scalameta trees only use `List`, conventionally we would pattern match this like `Pat.Var(name) :: Nil`